src/face_tracking/tracking_frame_processor.py

```
@@ -0,0 +1,78 @@
+        )
+        frame = image_drawer.draw_cross_hair_in_box(frame, face_trbl, 4, "green")
+
+    closest_box, closest_distance, closest_center = closest
+
+    vector_to_center = get_vector_xyz(frame_center_xyz, closest_center)
+
+    frame = image_drawer.draw_box(frame, closest_box, "red")
+    frame = image_drawer.draw_frame_center_cross_hair(frame, 2, 20, "red")
+
+    logger.debug(
+        f"Closest face at {frame_center_xyz, closest_center} with distance {closest_distance}"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)